### PR TITLE
XANAChain

### PR DESCRIPTION
**Pull request by SalmanNoborderz**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Reusing a chainID is problematic and only done in rare cases.
You will for sure need to add `"redFlags": ["reusedChainId"]`
And also show that the old one is not coming back.

**Comment by SalmanNoborderz** [comment]
@ligi the old chainId '8888' is of AmbrosChain Testnet of whom nothing works anymore, this is of XANAChain Mainnet, what proof will be required that it is not coming back?

**Comment by ligi** [comment]
e.g. a statement of them that they discontinued the network
but you will also need to add: "redFlags": ["reusedChainId"]

**Comment by SalmanNoborderz** [comment]
> e.g. a statement of them that they discontinued the network but you will also need to add: "redFlags": ["reusedChainId"]

@ligi nothing of theirs work anymore(RPC, explorers, twitter, telegram), their website is static with no links working, no links on CMC are working also...

I'll add "redFlags": ["reusedChainId"]

File: _data/chains/eip155-8888.json [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
   "chainId": 8888,
   "networkId": 8888,
+  "icon": "xeta",
   "explorers": [
     {
-      "name": "Ambros Chain Explorer",
-      "url": "https://testnet.ambrosscan.com",
-      "standard": "none"
+      "name": "XANAChain",
+      "url": "https://xanachain.xana.net/",
```
Comment (ligi) [inline _data/chains/eip155-8888.json:20]:
```suggestion
      "url": "https://xanachain.xana.net",
```

**Pull request merged**